src/utils/create-app-export.js

```diff
@@ -18,13 +23,43 @@ module.exports = function (options) {
   const { name: packageName } = JSON.parse(packageFile);
   const addonName = packageName || path.basename(destination);
 
-  const fileContent = `export { default } from '${addonName}/${fileOptions.type}/${fileName}';`;
+  const exportPath = `${addonName}/${fileOptions.type}/${fileName}`;
+  const fileContent = `export { default } from '${exportPath}';`;
 
   log(destPath);
   if (!dryRun) {
-    fse
+    return fse
       .outputFile(destPath, fileContent)
-      .then(() => {
+      .then(async () => {
+        if (updateImportsWhiteList.includes(fileType)) {
+          let { dir, name } = path.parse(sourceFile);
+          let sourcePath = path.join(dir, name);
+          const entityPath = `/${fileOptions.type}/`;
+          const [, filePath] = sourcePath.split(entityPath);
+          const oldImportPath = `${entityPath}${filePath}`;
+
+          log('\n Run the below command to update the imports');
+          log('----------------------------------- ');
+          log(
+            `\natam-codemod update-imports path/of/files/ --oldImportPath ${oldImportPath} --newImportPath ${exportPath}`
+          );
+
+          // Commenting it out as the ignore pattern and config options are not supported by codemod-cli.
+          // log(`\nUpdating the import paths matching ${oldImportPath} to ${exportPath}`);
+          let updateImports = false;
+          if (updateImports) {
```

Comment:
What are we doing here? the if condition will never be true here

Reply:
Updated the code